tonic/models/vic/grid_params.py

```diff
@@ -602,6 +732,71 @@ def __init__(self, organic_fract=False, spatial_frost=False,
                                'basin_depth': 'm',
                                'basin_area': 'fraction'}
 
+        # State Variables
+        self.state = {'gridcell': 'N/A',
+                      'Soil_moisture': 'mm',
+                      'Soil_ice': 'mm',
+                      'Canopy_water': 'mm',
+                      'Snow_age': 'time steps',
+                      'Snow_melt_state': '0 or 1',
+                      'Snow_coverage': 'fraction',
+                      'Snow_water_equivalent': 'mm',
+                      'Snow_surf_temp': 'deg C',
+                      'Snow_surf_water': 'mm',
+                      'Snow_pack_temp': 'deg C',
+                      'Snow_pack_water': 'mm',
+                      'Snow_density': 'kg/m3',
```

Comment:
I think we should standardize on the units strings following the [cf-conventions](http://cfconventions.org/Data/cf-standard-names/31/build/cf-standard-name-table.html).

This should be `'kg m-3'`.